MUI/src/main/java/mui/MUISetupProvider.java

```diff
@@ -212,4 +196,763 @@ public void setProgram(Program p) {
 	public JComponent getComponent() {
 		return mainPanel;
 	}
+
+	// RelativeLayout implementation from http://www.camick.com/java/source/RelativeLayout.java
```

Comment:
Is there any more information on this? What is the license? This should at least be in it's own file, if possible, and noted that we copied the implementation with the respective license

Reply:
there isn't any sembling of a licence other than the author saying it can be used and modified freely, but I think to avoid any complications I should be able to just achieve the same effect with the native GridBagLayout with a bit of effort

Reply:
implemented in aa6b84f, the UI should continue to work (and grow consistently, which was the reason I switched to that library in the first place)

![gridbaggrow (1)](https://user-images.githubusercontent.com/29654756/150789034-376dba36-39e6-4bd1-8459-27a64bbd194b.gif)